lib/travis/logs/app.rb

```diff
@@ -56,6 +57,21 @@ def initialize(existence = nil, pusher = nil)
       get "/uptime" do
         status 204
       end
+
+      post "/logs/:id/clear" do
```

Comment:
Yep, `DELETE` semantics are up to our interpretation.  It's not strictly a resource removal, although if we're allowing access to `/logs/:id` afterward and *not* returning 404, then I'd say going with `POST /logs/:id/clear` is probs better :+1:.